### PR TITLE
feature-3485 add tripId to the events-csv output

### PR DESCRIPTION
**Pull request by g-derzhavets**
Issue is done https://github.com/LBNL-UCB-STI/beam/issues/3485

Screenshot 2022-03-09 at 10 38 11

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3486)

### CODE REVIEW

**Comment by g-derzhavets** [comment]
Test!

**Comment by g-derzhavets** [comment]
Test!

**Comment by g-derzhavets** [comment]
Test!

**Comment by g-derzhavets** [comment]
Test!

**Comment by g-derzhavets** [comment]
Test!

File: src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PlanMerger.scala [hunk 1]
```diff
@@ -9,7 +9,7 @@ class PlanMerger(modeMap: Map[String, String]) extends Merger[InputPlanElement,
 
   private def transform(inputPlanElement: InputPlanElement): PlanElement = {
     PlanElement(
-      inputPlanElement.tripId.toString,
+      if (inputPlanElement.tripId.isEmpty) "" else inputPlanElement.tripId.get,
```
Comment (zneedell) [inline src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PlanMerger.scala]: Could this just be `getOrElse(tripId, "")`?

**Review by zneedell — APPROVED** [review]
One very minor suggestion but generally LGTM

File: src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PlanMerger.scala (hunk 1 above)
Comment (g-derzhavets) [reply src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PlanMerger.scala]: Agree, fixed, thanks!

**Comment by g-derzhavets** [comment]
Test!

**Pull request merged**